### PR TITLE
refactor: repo structure

### PR DESCRIPTION
**Pull request by Daanvdplas**
Separates the drink and ink-sandbox code from the opinionnated pop code. Reason for that is that we can easily push PR to the original repos and only have our own code separated from what can then be removed.

Reverted changes that were unnecessary in the drink code.

### CODE REVIEW

File: crates/pop-drink/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Pop Sandbox
```
Comment (chungquantin) [inline crates/pop-drink/README.md:1]: This document need to be updated I guess.

File: crates/pop-drink/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+- For `testnet` runtime
+
+Implementation of the sandbox runtime environment for `testnet` runtime located in `pop-node/runtime/testnet`
+
+```rs
+use pop_sandbox::TestnetSandbox;
+```
+
+- For `mainnet` runtime
```
Comment (chungquantin) [inline crates/pop-drink/README.md:33]: This is removed.

File: crates/pop-drink/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+#[drink::test(sandbox = Sandbox)]
+fn test(mut session: Session) {
+ // Your test case
+}
+```
+
+## Examples
+
+Please find more examples of `pop_drink` tests in the [`pop_drink/examples`](https://github.com/r0gue-io/pop-drink/tree/main/examples).
```
Comment (chungquantin) [inline crates/pop-drink/README.md:60]: No longer needed.

File: crates/pop-drink/README.md (hunk 1 above)
Comment (Daanvdplas) [reply crates/pop-drink/README.md:1]: This entire README need to be reworked but lets do that in a separate PR

Comment (Daanvdplas) [reply crates/pop-drink/README.md:1]: Created issue

**Pull request merged**